### ISSUE TITLE
Error during install if we don't have yarn

### ISSUE THREAD

**Issue by dhoko** [transcend-io/conflux#59]
:wave: 

The `npm install` is broken if we don't have yarn :/
Why do you install what's inside `example` cf https://github.com/ProtonMail/conflux/blob/master/package.json#L18 ? 

1. Clone the repo
2. npm i
3. :boom: 
or

1. Add the repo as a dep inside a project
2. npm i
3. :boom: 

```sh
│npm ERR! prepareGitDep > @transcend-io/conflux@2.0.2 prepare /home/dhoko/.npm/_cacache/tmp/git-clone-4487cc08
│npm ERR! prepareGitDep > cd example && yarn && cd ..
│npm ERR! prepareGitDep 
│npm ERR! prepareGitDep 
│npm ERR! prepareGitDep 2> npm WARN install Usage of the `--dev` option is deprecated. Use `--only=dev` instead.
│npm ERR! prepareGitDep npm WARN deprecated istanbul@0.4.5: This module is no longer maintained, try this instead:
│npm ERR! prepareGitDep npm WARN deprecated   npm i nyc
│npm ERR! prepareGitDep npm WARN deprecated Visit https://istanbul.js.org/integrations for other alternatives.
│npm ERR! prepareGitDep npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Pleas
│e, upgrade your dependencies to the actual version of core-js@3.
│npm ERR! prepareGitDep npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
│npm ERR! prepareGitDep npm WARN deprecated gulp-header@1.8.12: Removed event-stream from gulp-header
│npm ERR! prepareGitDep npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
│npm ERR! prepareGitDep npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
│npm ERR! prepareGitDep npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
│npm ERR! prepareGitDep sh: 1: yarn: not found
│npm ERR! prepareGitDep npm ERR! code ELIFECYCLE
│npm ERR! prepareGitDep npm ERR! syscall spawn
│npm ERR! prepareGitDep npm ERR! file sh
│npm ERR! prepareGitDep npm ERR! errno ENOENT
│npm ERR! prepareGitDep npm ERR! @transcend-io/conflux@2.0.2 prepare: `cd example && yarn && cd ..`
│npm ERR! prepareGitDep npm ERR! spawn ENOENT
│npm ERR! prepareGitDep npm ERR! 
│npm ERR! prepareGitDep npm ERR! Failed at the @transcend-io/conflux@2.0.2 prepare script.
│npm ERR! prepareGitDep npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
│npm ERR! prepareGitDep 
│npm ERR! prepareGitDep npm ERR! A complete log of this run can be found in:
│npm ERR! prepareGitDep npm ERR!     /home/dhoko/.npm/_logs/2020-05-18T12_37_53_314Z-debug.log
```

**Comment by jimmywarting** [comment]
Also wish to get rid of unrelated stuff from example like streamsaver, and replace yarn with npm

**Comment by bencmbrook** [comment]
This should be resolved by #66 (orig: #53)
